pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [build-system]
 requires = [
-    "Cython~=0.29.29",
+    "cython~=3.0.2",
```

Comment:
The package seems to build fine with cython 0, why is 3.0.2+ required?

Reply:
@adamjstewart I don't want to support different versions of Cython at the same time, that's all.

Reply:
Makes sense. It's not a big deal for pip but can be tricky for other package managers like spack. We don't yet have cython 3.0.2 packaged as it's quite new.